src/features/rewards/siteBanner/style.ts

```diff
@@ -6,11 +6,21 @@ import styled from 'styled-components'
 import { Props } from './index'
 import bg1Url from './assets/bg_bats.svg'
 import bg2Url from './assets/bg_hearts.svg'
+import palette from '../../../theme/colors'
 
 interface StyleProps {
   padding?: boolean
   bg?: string
   isTwitterTip?: boolean
+  monthly?: boolean | undefined
+}
+
+const getBgStyle = (monthly?: boolean) => {
+  if (!monthly) {
+    return `background: ${palette.blurple400}`
+  }
+
+  return `background-image: linear-gradient(180deg, #FF8907 0%, ${palette.orange500} 100%)`
```

Comment:
There isn't really a good theme substitute for the starter gradient value - all of the lighter oranges are too peachy 🍑

Reply:
sounds good, let's keep this one then